functions/public/Invoke-WPFUIElements.ps1

```diff
@@ -0,0 +1,321 @@
+function Invoke-WPFUIElements {
+    <#
+    .SYNOPSIS
+        Adds UI elements to a specified Grid in the WinUtil GUI based on a JSON configuration.
+    .PARAMETER configVariable
+        The variable/link containing the JSON configuration.
+    .PARAMETER targetGridName
+        The name of the grid to which the UI elements should be added.
+    .PARAMETER columncount
+        The number of columns to be used in the Grid. If not provided, a default value is used based on the panel.
+    .EXAMPLE
+        Invoke-WPFUIElements -configVariable $sync.configs.applications -targetGridName "install" -columncount 5
+    .NOTES
+        Future me/contributer: If possible please wrap this into a runspace to make it load all panels at the same time.
+    #>
+
+    param(
+        [Parameter(Mandatory)]
+        [PSCustomObject]$configVariable,
+
+        [Parameter(Mandatory)]
+        [string]$targetGridName,
+
+        [Parameter(Mandatory)]
+        [int]$columncount
+    )
```

Comment:
```suggestion
    param(
        [Parameter(Mandatory, position=0)]
        [PSCustomObject]$configVariable,

        [Parameter(Mandatory, position=1)]
        [string]$targetGridName,

        [Parameter(Mandatory, position=2)]
        [int]$columncount
    )
```
Making Positional Parameters forced in a specific order is better if you can (and I think positional parameters will work even if you don't specify it.. but this'll force it in a desired order)